src/templates/blog-post.js

```diff
@@ -68,10 +68,23 @@ const BlogPostTemplate = ({ data, pageContext, location }) => {
   );
 };
 
+BlogPostTemplate.propTypes = {
+  data: shape({
+    site: object,
+    post: object,
+    author: object,
+  }),
+  pageContext: shape({
+    previous: object,
+    next: object,
+  }),
+  location: object,
```

Comment:
Isn't it better to create an specific object/model for every prop so everyone know the detail of that prop?